index.html

```diff
@@ -25,6 +25,20 @@ <h1 id="logo">Aziz Mejri</h1>
           </li>
         </ul>
       </nav>
+
+      <nav id="mobile-menu-container">
+        <div class="menu-shape">
+          <img class="img-square" src="./media/svg/square.svg" alt="square" />
+          <i id="mobile-close-menu" class="fa fa-times"></i>
```

Comment:
It seems you went for completely different style!
I would kindly suggest that you try to follow the original template as close as you can!
# Original
![image](https://user-images.githubusercontent.com/28903404/139600280-6dc37c16-51cb-40e0-bfb1-5a993f40f8e4.png)

# Yours
![image](https://user-images.githubusercontent.com/28903404/139600294-096d3714-3a6f-47de-97e5-1d51413d73a6.png)

Remember it is very important to stick to the original template! You will have a chance to show your creativity in the upcoming projects! But for now I kindly suggest you change the style of the close menu button!